src/repositories/component/component-complex.repository.ts

```
@@ -0,0 +1,301 @@
+  }
+
+  /**
+   * Create dependency relationship between components
+   */
+  private async createDependencyRelationship(
+    logicalRepositoryName: string,
+    componentBranch: string,
+    componentId: string,
+    depId: string,
+  ): Promise<void> {
+    const graphUniqueId = this.createGraphUniqueId(logicalRepositoryName, componentBranch, componentId);
```

ESLint

[error] Replace `logicalRepositoryName,·componentBranch,·componentId` with `⏎······logicalRepositoryName,⏎······componentBranch,⏎······componentId,⏎····`

(prettier/prettier)